src/ServiceControl.DockerImages/ServiceControl.DockerImages.csproj

```diff
@@ -90,20 +92,44 @@
     <!-- monitoring instance and sidecar dockerfiles -->
     <Copy SourceFiles="servicecontrol.transport.monitoring.init-windows.dockerfile-template" DestinationFiles="$(DockerfilesFolder)servicecontrol.%(SupportedTransport.DockerfileCustomization).monitoring.init-windows.dockerfile" />
     <Copy SourceFiles="servicecontrol.transport.monitoring-windows.dockerfile-template" DestinationFiles="$(DockerfilesFolder)servicecontrol.%(SupportedTransport.DockerfileCustomization).monitoring-windows.dockerfile" />
+
+    <ItemGroup>
```

Comment:
Since this target is called `ExpandDockerfileTemplates`, I think it makes more sense to create another template for copying the files out of the deploy folder instead of adding it to this one.

Reply:
Done in 67263e063d671bd54ac2500f53e2fb97989021d5